azurerm/internal/services/keyvault/key_vault_certificate_resource.go

```diff
@@ -474,20 +474,21 @@ func resourceKeyVaultCertificateCreate(d *pluginsdk.ResourceData, meta interface
 
 		log.Printf("[DEBUG] Waiting for Key Vault Certificate %q in Vault %q to be provisioned", name, *keyVaultBaseUrl)
 		stateConf := &pluginsdk.StateChangeConf{
-			Pending:    []string{"Provisioning"},
-			Target:     []string{"Ready"},
-			Refresh:    keyVaultCertificateCreationRefreshFunc(ctx, client, *keyVaultBaseUrl, name),
-			MinTimeout: 15 * time.Second,
-			Timeout:    d.Timeout(pluginsdk.TimeoutCreate),
+			Pending:      []string{"Provisioning"},
+			Target:       []string{"Ready"},
+			Refresh:      keyVaultCertificateCreationRefreshFunc(ctx, client, *keyVaultBaseUrl, name),
+			PollInterval: 10 * time.Second,
+			MinTimeout:   15 * time.Second,
```

Comment:
`PollInterval` overrides `MinTimeout` so we should probably make this:
```suggestion
			PollInterval: 15 * time.Second,
```